jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -509,47 +509,6 @@ export class MountPlugin implements IMountPlugin {
     this._saveInputSpecSignal.emit(this._repoViewInputSpec);
   };
 
-  restoreMountedReposList = async (): Promise<void> => {
```

Comment:
Just curious why this test specifically is being removed?

Reply:
this isn't a test, this is a method that is now obsolete. since we maintain two separate file managers for explore view and for datum view respectively, there is no state to restore unlike with mount-server. `saveMountedReposList` could probably be pruned a bit too but at least some of the functionality there is still in use, e.g. generating an initial input spec for datum view from the mounted repos in explore.